CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## WHAT'S NEW
+
+##Version 1.0.2
+- Bug fixes 
+- Performance improvement
```

Comment:
This is too generic

Reply:
@smaranjitghose I have made some changes.
Do review.
Thank You.